pkg/imageupdater/airflow.go

```diff
@@ -53,3 +72,59 @@ func (c *client) updateAirflowBaseImage(ctx context.Context) (bool, error) {
 
 	return false, nil
 }
+
+func (c *client) updateGlobalEnvs(ctx context.Context) (bool, error) {
+	globalEnvsSQL, err := c.repo.GlobalValueGet(ctx, gensql.ChartTypeAirflow, airflowGlobalEnvKey)
+	if err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
+			return false, nil
+		}
+		return false, err
+	}
+
+	type globalEnv struct {
+		Name  string `json:"name"`
+		Value string `json:"value"`
+	}
+	globalEnvs := []*globalEnv{}
+	if err := json.Unmarshal([]byte(globalEnvsSQL.Value), &globalEnvs); err != nil {
+		return false, err
+	}
+
+	globalEnvsUpdated := false
+	for _, env := range globalEnvs {
+		if contains(imageEnvNames, env.Name) {
+			imageParts := strings.Split(env.Value, ":")
+			if len(imageParts) != 2 {
+				return false, fmt.Errorf("invalid image format for image %v, should be <image>:<tag>", env.Value)
+			}
+			garImage, err := getLatestImageInGAR(imageParts[0], "")
+			if err != nil {
+				return false, err
+			}
+
+			if imageParts[1] != garImage.Tag {
+				env.Value = fmt.Sprintf("%v:%v", imageParts[0], garImage.Tag)
```

Comment:
```suggestion
			currentImageParts := strings.Split(env.Value, ":")
			if len(currentImageParts) != 2 {
				return false, fmt.Errorf("invalid image format for image %v, should be <image>:<tag>", env.Value)
			}

			latestImage, err := getLatestImageInGAR(currentImageParts[0], "")
			if err != nil {
				return false, err
			}

			if currentImageParts[1] != latestImage.Tag {
				env.Value = fmt.Sprintf("%v:%v", currentImageParts[0], latestImage.Tag)
```

I mine øyne er `latest` og `current` lettere å lese og skjønne.